README.md

```diff
@@ -92,6 +93,7 @@ MLServer to start serving your machine learning models.
 - [Serving a `tempo` pipeline](./docs/examples/tempo/README.md)
 - [Serving a custom model](./docs/examples/custom/README.md)
 - [Serving an `alibi-detect` model](./docs/examples/alibi-detect/README.md)
+- [Serving a `HuggingFace` model](./docs/examples/huggingface/README.m)
```

Comment:
Is there a missing `d` in `.md`? Not sure if Sphinx is happy with that.

@saeid93 would you be able to test it out on the RTD preview for this PR?

https://mlserver--719.org.readthedocs.build/en/719/

Reply:
Please check now. I think it should be fine. Testing locally looks fine.